lib/logger.js

```diff
@@ -12,9 +12,13 @@ var _util = require('./util');
 function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
 
 var os = require('os');
-var _ = require('underscore');
-var request = require('request');
+
+var _require = require('lodash'),
```

Comment:
typo?

Reply:
it is auto generated code by babel =)

Reply:
ok... ugly

Reply:
well, because we have such target js version